cmd/zoekt-sourcegraph-indexserver/queue_test.go

```diff
@@ -126,20 +125,42 @@ func TestQueue_Bump(t *testing.T) {
 }
 
 func TestQueue_Integration_DebugQueue(t *testing.T) {
+
 	// helper function to normalize the queue's debug output - this makes the test less brittle
 	// + makes it much less annoying to make edits to the expected output in a way that doesn't
 	// materially affect the caller
 	normalizeDebugOutput := func(output string) string {
-		// remove any leading or trailing whitespace
 		output = strings.TrimSpace(output)
 
-		// remove trailing whitespace at the of each row (could come from column padding)
-		var lines []string
-		for _, l := range strings.Split(output, "\n") {
-			lines = append(lines, strings.TrimRightFunc(l, unicode.IsSpace))
+		var outputLines []string
+		for i, line := range strings.Split(output, "\n") {
+			// remove any leading or trailing whitespace at the end of each row (column padding)
+			line = strings.TrimSpace(line)
```

Comment:
do you still need this if you use strings.Fields?

Reply:
You are correct! I wasn't sure what the output would be you had leading spaces (ex: "    hello keegan " -> [" ", "hello", "keegan"]), but this seems to work.